src/udp.c

```diff
@@ -133,6 +132,46 @@ static int udp_input(const struct ip_hdr *ip_hdr,
 }
 IP_PROTO_INPUT_HANDLER(IP_PROTO_UDP, udp_input);
 
+static uint16_t udp_csum(const void *buff,
```

Comment:
Rename to `udp_checksum` for readability. Also, ensure to indent with `clang-format`.